test/modules/testModule/Module.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function ModuleName(options,UI) {
```

Comment:
Let's call it `TestModule` just to be clear. Awesome!

Reply:
done!!